### PR TITLE
Rename to Harness Terraform

### PR DESCRIPTION
**Pull request by nassergonzalez**
Removing the different plugins that were just references to the same plugin
Renaming plugin name to harness terraform
Fixing the image repository to point to the proper image

### CODE REVIEW

File: plugins/harness-terraform/content.yaml [hunk 1]
```diff
@@ -1,17 +1,16 @@
...
+title: Harness Terraform
+author: Harness
 tags:
   - infrastructure
   - harness
   - terraform
 logo: terraform.svg
 repo: https://github.com/drone-plugins/drone-terraform
-image: https://hub.docker.com/r/plugins/drone-terraform
+image: https://hub.docker.com/r/plugins/harness_terraform:linux-amd64
```
Comment (d1wilko) [inline plugins/harness-terraform/content.yaml:9]: can we change this to just `https://hub.docker.com/r/plugins/harness_terraform`

File: plugins/harness-terraform/content.yaml [hunk 2]
```diff
@@ -1,17 +1,16 @@
...
 license: None
-readme: https://github.com/drone-plugins/drone-terraform
+readme: https://github.com/drone-plugins/drone-terraform/tree/main#readme
```
Comment (d1wilko) [inline plugins/harness-terraform/content.yaml:11]: can we change this to just `image: https://github.com/drone-plugins/drone-terraform/blob/main/README.md`

File: plugins/harness-terraform/content.yaml [hunk 3]
```diff
@@ -20,12 +19,12 @@ example: |
...
-        operation: plan
+        command: apply
 
 properties:
-  operation:
+  command:
     type: string
-    defaultValue: "plan"
-    description: Command that indicates what action the plugin will perform
+    defaultValue: 'init'
```
Comment (d1wilko) [inline plugins/harness-terraform/content.yaml:27]: can you use "'s here :)

**Review by d1wilko — CHANGES_REQUESTED** [review]
few small changes then we can merge :)

File: plugins/harness-terraform/content.yaml (hunk 3 above)
Comment (nassergonzalez) [reply plugins/harness-terraform/content.yaml:27]: Resolved

**Review by d1wilko — APPROVED** [review]
lgtm :)

**Pull request merged**